packages/core/src/invoke.ts

```diff
@@ -4,19 +4,26 @@ import {
   Subscribable,
   BehaviorCreator,
   SCXML,
-  InvokeMeta
-} from '.';
+  InvokeMeta,
+  ActorRef
+} from './types';
+import { State } from './State';
 
-import { isFunction, mapContext } from './utils';
+import { isFunction, mapContext, warn } from './utils';
 import { AnyEventObject } from './types';
 import { MachineNode } from './MachineNode';
 import {
   createMachineBehavior,
-  createCallbackBehavior,
+  createDeferredBehavior,
   Behavior,
   createObservableBehavior,
   createPromiseBehavior
 } from './behavior';
+import { InvokeActionObject } from '../dist/xstate.cjs';
```

Comment:
this should be imported from `./types` or something `/dist/` should never appear in our internal imports

Reply:
I'm not sure why VS Code sometimes auto-imports from `/dist/` - this only occurs in `next`.